D1371R0.pandoc

```diff
@@ -1135,70 +1146,40 @@ inspect (v) {
 }
 ```
 
-In the example above `^` is used to make former pattern to look for name `x`
+In the example above, `^` is used to make former pattern to look for name `x`
```

Comment:
It seems simpler to me to just refer to the patterns directly.

e.g., "In the above example, `^x` performs unqualified name look up for `x`, while `x` introduces a new identifier `x` in scope"